### PR TITLE
Serialize floats in restore

### PR DESCRIPTION
**Pull request by benrudolph**
@TylerSheffels http://manage.dimagi.com/default.asp?180581#1026480

### CODE REVIEW

**Comment by orangejenny** [comment]
looks good, will wait on tests

File: corehq/apps/fixtures/models.py [hunk 1]
```diff
@@ -268,16 +269,10 @@ def type_check(self):
             raise FixtureTypeCheckError("fields %s from fixture data %s not in fixture data type" % (', '.join(fields), self.get_id))
 
     def to_xml(self):
-        def _serialize(val):
-            if isinstance(val, (int, Decimal)):
```
Comment (benrudolph) [inline corehq/apps/fixtures/models.py]: just an fyi, the fix was actually adding `float` to this list

**Comment by czue** [comment]
nice

**Comment by benrudolph** [comment]
bleh, highly doubtful that the test failure is real. will test locally

**Comment by czue** [comment]
definitely not real.

**Comment by czue** [comment]
I suspect https://github.com/dimagi/commcare-hq/pull/8533

**Comment by benrudolph** [comment]
thanks, will merge master

**Comment by benrudolph** [comment]
hmm, if tests don't rerun on merges what's the process to get travis going again?

**Comment by czue** [comment]
looks like you figured it out.

**Pull request merged**